lib/YaST/Firewall/MainPage.pm

```diff
@@ -0,0 +1,58 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Handles Firewall Main Page(Include left view tree menu and bottom button).
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package YaST::Firewall::MainPage;
+use strict;
+use warnings;
+
+sub new {
+    my ($class, $args) = @_;
+    my $self = bless {
+        app => $args->{app}
+    }, $class;
+    return $self->init();
+}
+
+sub init {
+    my $self = shift;
+    $self->{tre_overview_tree} = $self->{app}->tree({id => "\"Y2Firewall::Widgets::OverviewTree\""});
```

Comment:
s/tre_overview_tree/tre_overview/

Reply:
The Widgets name is "OverviewTree"
so base our rule we use extra prefix "tre" + "overview_tree"(name of widgets)

Reply:
It doesn't say how to name the second part, I think it is open because, we don't use the id, we just make up a name that is reasonable, so we don't call it `tre_y2firewall_widgets_overviewtree`, that is because those id contains strange strings which don't have much use as name of variable, so in this case with `tree_overview` we don't repeat the 'tree' part.